multithreading/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AsyncSort
```

Comment:
Good try. We will get there. Got to understand the basics before we start in on async and parallel stuff.

Reply:
yea im still a newb ha ha but i learned A LOT while trying to do this.. so least i got a little out of it.